packages/manager/apps/dedicated/client/app/app.less

```diff
@@ -57,4 +57,8 @@
   #licenseOrderBlockFilters {
     cursor: pointer;
   }
+
+  .overflow-visible {
+    overflow: visible !important;
+  }
```

Comment:
**nitpick:** Can you please use the scoped css classes. It is not recommended to define a CSS class required for a ips in the CSS file dedicated for the app.